.github/workflows/handle.yml

```diff
@@ -109,3 +108,85 @@ jobs:
           else
             echo "::notice title=CLI Request Handle::Skip because tool.yaml is not found"
           fi
+
+      - name: 'IAM Request Comment'
+        if: '${{ always()}}'
+        uses: 'actions/github-script@98814c53be79b1d30f795b907e553d8679345975' # ratchet:actions/github-script@v6
+        with:
+          github-token: '${{ github.token }}'
+          retries: '3'
+          script: |+
+            var req;
+            const fs = require("fs");
+            const outcome = '${{ steps.handle_iam.outcome }}';
+            switch (outcome) {
+              case 'success':
+                req = fs.readFileSync(
+                  `iam.yaml`,
+                  { encoding: "utf8" }
+                );
+
+                await github.rest.issues.createComment({
+                  owner: context.repo.owner,
+                  repo: context.repo.repo,
+                  issue_number: ${{ github.event.pull_request.number }},
+                  body: `**\`Access on Demand\`** - 🟩 **\`IAM\`** Completed request.
+
+            <details>
+            <summary>Details</summary>
+            Added below IAM permissions, and they will be expired in ${{ env.LABELED_DURATION || env.DEFAULT_DURATION }}.
+
+            \`\`\`
+            ${req}
+            \`\`\`
+            </details>
```

Comment:
If we're sure the file exists, we can output the req first and then output whether it succeeded or failed (instead of duplicating the code)

Reply:
if we are sure the file exists, but I think a comment to people that no request is handled is also useful.